config/config.yaml

```diff
@@ -173,4 +173,14 @@ analysis:
         # 'euclidean', 'manhattan', 'cosine'
         metric: 'euclidean'
       evaluation:
+        # evaluation per dataset-goldstandard pair
+        # evalution will not run unless ml include is set to true
         include: true
+        # adds evaluation per algorithm per dataset-goldstandard pair
+        # evalution per algortihm will not run unless ml include and ml aggregate_per_algorithm is set to true
+        aggregate_per_algorithm: true
+        # TODO: should we decouple parts of eval that involve ml
```

Comment:
Lots of coupling happening now. I put in a solution for now in config.py, but is it worth separating the functions into their own true/ false?

Reply:
Maybe deal with some of the coupling by giving warnings and stopping the flow rather than silently shutting things off